db/seeds.d/02-roles_list.rb

```diff
@@ -38,38 +40,7 @@ def role_names
     end
 
     def base_manage_permissions
-      [:create_architectures, :edit_architectures, :destroy_architectures,
-       :create_authenticators, :edit_authenticators, :destroy_authenticators,
-       :create_bookmarks, :edit_bookmarks, :destroy_bookmarks,
-       :create_compute_resources, :edit_compute_resources, :destroy_compute_resources,
-       :create_compute_resources_vms, :edit_compute_resources_vms, :destroy_compute_resources_vms, :power_compute_resources_vms, :console_compute_resources_vms,
-       :create_domains, :edit_domains, :destroy_domains,
-       :create_realms, :edit_realms, :destroy_realms,
-       :create_environments, :edit_environments, :destroy_environments, :import_environments,
-       :create_external_variables, :edit_external_variables, :destroy_external_variables,
-       :create_external_parameters, :edit_external_parameters, :destroy_external_parameters,
-       :create_hostgroups, :edit_hostgroups, :destroy_hostgroups,
-       :create_hosts, :edit_hosts, :destroy_hosts, :build_hosts, :power_hosts, :console_hosts, :ipmi_boot_hosts, :puppetrun_hosts,
-       :edit_classes,
-       :create_params, :edit_params, :destroy_params,
-       :create_images, :edit_images, :destroy_images,
-       :create_locations, :edit_locations, :destroy_locations, :assign_locations,
-       :create_media, :edit_media, :destroy_media,
-       :create_models, :edit_models, :destroy_models,
-       :create_operatingsystems, :edit_operatingsystems, :destroy_operatingsystems,
-       :create_provisioning_templates, :edit_provisioning_templates, :destroy_provisioning_templates, :deploy_provisioning_templates,
-       :create_ptables, :edit_ptables, :destroy_ptables,
-       :create_puppetclasses, :edit_puppetclasses, :destroy_puppetclasses, :import_puppetclasses,
-       :create_smart_proxies, :edit_smart_proxies, :destroy_smart_proxies,
-       :create_smart_proxies_autosign, :destroy_smart_proxies_autosign,
-       :edit_smart_proxies_puppetca, :destroy_smart_proxies_puppetca,
-       :create_ssh_keys, :destroy_ssh_keys,
-       :create_subnets, :edit_subnets, :destroy_subnets, :import_subnets,
-       :create_usergroups, :edit_usergroups, :destroy_usergroups,
-       :create_users, :edit_users, :destroy_users,
-       :destroy_config_reports, :upload_config_reports,
-       :upload_facts,
-       :create_trends, :edit_trends, :destroy_trends, :update_trends]
+      PermissionsList.permissions.reject { |resource, name| name.start_with?('view_') }.map(&:last) - manage_organizations_permissions
```

Comment:
there's `map(&:last)` to get only permissions, I'll add `to_sym` and convert it to block